### PR TITLE
WIP: feature: sync futures data and backtest with them

### PR DESCRIPTION
**Pull request by andycheng123**
tables for futures data
sync futures data
backtest with futures data

### CODE REVIEW

**Review by c9s — APPROVED** [review]

File: pkg/backtest/exchange.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ package backtest
 import (
 	"context"
 	"fmt"
+	exchange2 "github.com/c9s/bbgo/pkg/exchange"
```
[the remark this answers is not in the document]
Comment (andycheng123) [reply pkg/backtest/exchange.go:33]: goland added that. i hav no idea

[the remark this answers is not in the document]
Comment (andycheng123) [reply pkg/backtest/exchange.go]: removed

File: pkg/backtest/exchange.go [hunk 2]
```diff
@@ -381,7 +381,20 @@ func (e *Exchange) SubscribeMarketData(
 		intervals = append(intervals, interval)
 	}
 
-	log.Infof("querying klines from database with exchange: %v symbols: %v and intervals: %v for back-testing", e.Name(), symbols, intervals)
+	//_, isFutures, _, _ := exchange2.GetSessionAttributes(e.publicExchange)
```
Comment (c9s) [inline pkg/backtest/exchange.go:384]: remove unused code?

**Review by c9s — APPROVED** [review]

File: pkg/backtest/exchange.go (hunk 2 above)
Comment (andycheng123) [reply pkg/backtest/exchange.go]: done

**Pull request merged**